.travis.yml

```diff
@@ -0,0 +1,2 @@
+language: node_js
+script: yarn danger ci
```

Comment:
@orta I see that in https://github.com/artsy/force/pull/3061#issuecomment-436480774 you mentioned that danger got removed from CI, but can you elaborate?  I'm just wondering if there's a way that we can modify the configuration for Circle than introduce another build system for this project.

Reply:
Yep, could be done, but it's simpler this way as it doesn't have to run inside docker and Travis is more reliable than Circle for Danger (due to the way they both handle ENV vars)

https://github.com/danger/danger-js/blob/f648c8a9d9e0bf1710bc6de9b78ef6fe54829066/source/ci_source/providers/Circle.ts#L4-L23

That said, this could also be ported to peril (instead of looking in the node modules, pull out the GraphQL schema from the git repo) but this should be enough for now.

Reply:
Hah, nice, looks like that is the way to go - the graphql schema for reaction was removed last week, https://github.com/artsy/reaction/pull/1631